02-KernelModules_CharacterDevice/Create_device.h

```diff
@@ -0,0 +1,94 @@
+#pragma once
```

Comment:
Using #pragma once should work on any modern compiler, but I don't see any reason not to use a standard #ifndef include guard. It works just fine. The one caveat is that GCC didn't support #pragma once before version 3.4.